### PR TITLE
#6254 hover issue

### LINKED ISSUE #6254: Weird hover/default background
image
image

### PR DESCRIPTION
**Pull request by prachi00**
a**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Closes #6254
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)

## Screenshot 📸

- [x] My fix has changed UI

Screen Shot 2023-06-17 at 4 54 13 PM

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
Hello, something strange:
- Sidebar content should work with `$sidebar-content-background-color:`
- Search was working well before, has something changes regarding oruga version? or maybe I made some changes in `NeoAutocomple` that is making this result

**Comment by prachi00** [comment]
Screen Shot 2023-06-18 at 2 17 53 PM
@roiLeo for search, this css was added 3 days ago that broke it

**Review by roiLeo — COMMENTED** [review]
https://github.com/kodadot/nft-gallery/assets/9987732/b036abe8-58b8-4bc1-baf6-47c1339e7614

Hover still look weird, I'll check if I can do something

**Review by roiLeo — COMMENTED** [review]
Found something for Sidebar:
- code is duplicated
Capture d’écran 2023-06-19 à 8 53 35 AM

which means something wrong with imports.
When I remove this line https://github.com/kodadot/nft-gallery/blob/ce5ab4054216fffab38926cceea3fc8385aca562/styles/index.scss#L11 it work like a charm but something else can break

**Comment by prachi00** [comment]
> Found something for Sidebar:
> 
> * code is duplicated
> 
> Capture d’écran 2023-06-19 à 8 53 35 AM
> which means something wrong with imports. When I remove this line
> 
> https://github.com/kodadot/nft-gallery/blob/ce5ab4054216fffab38926cceea3fc8385aca562/styles/index.scss#L11
> 
> it work like a charm but something else can break

not sure why though

**Comment by roiLeo** [comment]
> not sure why though

Because we import oruga ui stylesheet two times, one form `libs/ui` and the other one from `styles/index.scss`.
But the one taking in count is `index.scss` so we can't override variable from `libs/ui`

**Review by roiLeo — COMMENTED** [review]
Hello, could you remove your changes in `NeoSidebar`, color issue is now fixed

only remaining is searchBar hover

**Comment by roiLeo** [comment]
> Enregistrement.de.l.ecran.2023-06-19.a.8.23.27.AM.mov
> 
> Hover still look weird, I'll check if I can do something

I still have this problem

**Comment by exezbcz** [comment]
can confirm 
image

**Comment by prachi00** [comment]
> can confirm image

@exezbcz @roiLeo fixed

**Review by roiLeo — COMMENTED** [review]
Hover issue comes from this part of code https://github.com/kodadot/nft-gallery/blob/280a1f150e341b68831f8595758a24bce3ff4ec6/libs/ui/src/components/NeoInputitems/NeoInputitems.scss#L43-L50

**Comment by prachi00** [comment]
> Hover issue comes from this part of code
> 
> https://github.com/kodadot/nft-gallery/blob/280a1f150e341b68831f8595758a24bce3ff4ec6/libs/ui/src/components/NeoInputitems/NeoInputitems.scss#L43-L50

yes but we can remove this right, as its probably being used somewhere

File: styles/components/_search.scss [hunk 1]
```diff
@@ -141,6 +141,15 @@
...
     }
   }
+
+  .o-acp__item {
+    padding: 0;
+    @include ktheme() {
+      background: theme('background-color');
+      color: theme('text-color')!important;
+
+    }
```
Comment (Jarsen136) [inline styles/components/_search.scss:151]:
```suggestion
      background: unset;
      color: unset;
```
How about this? I guess we do not need to set colour again here.

**Review by Jarsen136 — APPROVED** [review]

**Review by preschian — APPROVED** [review]

File: styles/components/_search.scss [hunk 2]
```diff
@@ -141,6 +141,14 @@
       padding: 0px;
     }
   }
+
+  .o-acp__item {
+    padding: 0;
+    @include ktheme() {
```
Comment (roiLeo) [inline styles/components/_search.scss:147]: why include ktheme if you don't use `theme` ?

**Review by roiLeo — APPROVED** [review]
> > Hover issue comes from this part of code
> > https://github.com/kodadot/nft-gallery/blob/280a1f150e341b68831f8595758a24bce3ff4ec6/libs/ui/src/components/NeoInputitems/NeoInputitems.scss#L43-L50
> 
> yes but we can remove this right, as its probably being used somewhere

I would move this part up (under `o-inputit__container` class)

otherwise wfm ✅

File: styles/components/_search.scss (hunk 2 above)
Comment (vikiival) [reply styles/components/_search.scss:147]: cc @prachi00

**Comment by vikiival** [comment]
@exezbcz is it oki?

**Review by roiLeo — CHANGES_REQUESTED** [review]
Capture d’écran 2023-06-27 à 8 54 25 AM

missing `!important` on `background: unser;`

**Comment by prury** [comment]
Darkmode still needs fixing

**Comment by prachi00** [comment]
fixed

**Comment by prury** [comment]
Lines under tabs are not showing: 
image
image

i think it should at least resemble this one: 
image

or have the same color of the other lines

**Comment by prachi00** [comment]
> Lines under tabs are not showing: image image
> 
> i think it should at least resemble this one: image
> 
> or have the same color of the other lines

was it there before? I dont see it in canary too, so probably not related to this PR 
Screen Shot 2023-06-27 at 5 39 22 PM

**Comment by prury** [comment]
> was it there before? I dont see it in canary too, so probably not related to this PR <img alt="Screen Shot 2023-06-27 at 5 39 

From Canary: 
image

**Comment by prachi00** [comment]
Screen Shot 2023-06-27 at 6 43 44 PM
weird I see it, on chrome and firefox

**Comment by roiLeo** [comment]
> weird I see it, on chrome and firefox

we should have a line under active tab, but it's not present anymore since we lost some part..

**Review by roiLeo — APPROVED** [review]

**Pull request merged**